Mastodon/Scene/HomeTimeline/HomeTimelineViewModel+LoadLatestState.swift

```diff
@@ -132,6 +132,17 @@ extension HomeTimelineViewModel.LoadLatestState {
                         query: .init(local: true, sinceID: sinceID),
                         authenticationBox: viewModel.authContext.mastodonAuthenticationBox
                     )
+                case let .list(id):
+                    response = try await viewModel.context.apiService.listTimeline(
+                        id: id,
+                        query: .init(sinceID: sinceID),
```

Comment:
More of a gut feeling:

In case of a `timelineContext` switch (from `home` to `public` or `hashtag`, for example), does it make sense to reset `sinceID` explicitly or is it enough just to reset `viewModel.dataController.records = []` ?

Reply:
We don't need to manually reset it as it will be computed upon entering the new loading state.